rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/widget/CovidRDTBarcodeFactory.java

```diff
@@ -57,6 +63,8 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 
     protected abstract String[] splitCSV(String barcodeCSV);
 
+    protected void sensorTriggered() {};
```

Comment:
This method is only relevant in the `OneScanBarcodeFactory` that detects the sensor, no need to have it here.

Reply:
fixed.